Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageContentView_Tests.swift

```diff
@@ -93,6 +93,30 @@ final class ChatMessageContentView_Tests: XCTestCase {
         // Assert message content view is rendered correctly.
         AssertSnapshot(view)
     }
+    
+    func test_appearanceForSystemMessage() {
+        // Create a system message
+        let systemMessage: ChatMessage = .mock(
+            id: .unique,
+            cid: .unique,
+            text: "Some member was added",
+            type: .system,
+            author: myFriend,
```

Comment:
out of curiosity, who is set as an author for a system message? 🤔 I'd expect the `author == nil` but the field in non-optional

Reply:
The `author` of a system message is not nil. In this case, it is the creator of the channel